src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -72,6 +72,16 @@ export class FilterTimestamp extends HTMLElement {
   };
 
   connectedCallback() {
+    const searchParams = new URLSearchParams(
+      document.location.search.substring(1)
+    );
+
+    const filterTabId = searchParams.get('filterTabId');
+    // context menu doesn't work on devtool_page
+    if (filterTabId) {
+      return;
+    }
```

Comment:
To disable the context menu while running in a devtools extension context we may use a different approach, e.g. one of the following ones should work:

- check if `browser.menus` and `browser.menus.onClicked` are defined, and early exit if they are not
- or check if `browser.devtools` is defined  

The first approach may be better, because it does runtime detection for the feature that are going to be needed for this component to be able to work as expected.

This way we can keep the "filter by timestamp" context menu is we open the activitylog page in the same mode (filtered by tab id) in an extension context where the context menu API is available.

Also, add a mention to the issue that is tracking the issue in the inline comment.

Reply:
done. Thanks :-)